src/index.html

```diff
@@ -13,7 +13,7 @@ <h1>Leaderboard</h1>
     <h2>Recent scores</h2>
     <input class="refBtn" id="refId" type="button" value="Refresh">
     <div class="scores" >
-        <table id="sc">
+        <table id="sc" class="scores">
             

```

Comment:
- [ ] [OPTIONAL] I appreciate your effort working on the design 🥇,  however, it'll be better to fix your margins, and padding, to have a good user interface 👍 

![Screenshot 2023-07-27 142158](https://github.com/faisalakb/leaderboardAPI/assets/57408419/d41645ce-9839-4a1b-ba2b-39a8834f8610)